crypto/src/ed25519.rs

```diff
@@ -3,7 +3,12 @@
 use base64ct::{Base64, Encoding};
 use ed25519_consensus::{batch, VerificationKeyBytes};
 use once_cell::sync::OnceCell;
-use serde::{de, Deserialize, Serialize};
+use serde::{
+    de::{self, MapAccess, SeqAccess, Visitor},
+    ser::SerializeStruct,
+    Deserialize, Serialize,
+};
+use serde_bytes::{ByteBuf, Bytes};
```

Comment:
Nit (that you can ignore): we already use `serde_with` elsewhere, and it has analogous byte types. But that may be a bit too much OCD on my part.
https://docs.rs/serde_with/latest/serde_with/

Reply:
Thanks for the suggestion. `serde_with::Bytes` seems to be missing some features compared to `serde_bytes::Bytes`, e.g. wrapping `&[u8]`. It is a bit hard to use it in the situation here.